.github/workflows/generate-summary.yaml

```diff
@@ -68,18 +68,36 @@ jobs:
           key: submodules-archive-9 # Numbered archive to allow for easy transition when bumping submodules
 
       - name: Initialize gcc
-        if: steps.retrieve-cache.outputs.cache-hit != 'true'
+        if: ${{ steps.retrieve-cache.outputs.cache-hit != 'true' && github.workflow != 'Run-Frequent-Coordination' }}
         id: gcc-cache
         uses: ./.github/actions/common/init-and-pull-gcc
         with:
           init: true
+          branch: master
 
         # Does not remove and reclone gcc if we hit cache
       - name: Checkout GCC
-        if: steps.gcc-cache.outcome == 'skipped'
+        if: ${{ steps.gcc-cache.outcome == 'skipped' && github.workflow != 'Run-Frequent-Coordination' }}
         uses: ./.github/actions/common/init-and-pull-gcc
         with:
           init: false
+          branch: master
```

Comment:
Can we pass in the branch similar to what we do for init-submodules? That way we can skip all these github.workflow checks

Reply:
I changed the github.workflow checks to inputs.prefix checks instead.

Reply:
If we have a branch variable, we can use that directly and avoid duplicating the initialize/checkout gcc actions

Reply:
updated